src/Middleware/Rewrite/src/RewriteOptionsExtensions.cs

```diff
@@ -76,7 +76,7 @@ public static RewriteOptions AddRedirect(this RewriteOptions options, [StringSyn
     }
 
     /// <summary>
-    /// Redirect a request to https if the incoming request is http, with returning a 301
+    /// Redirect a request to https if the incoming request is http, with returning a 301 - Moved Permanently status code.
     /// status code for permanently redirected.
```

Comment:
> status code for permanently redirected.

This one already says it sets 301 meaning permanently redirected.

Reply:
Yes, `Also then tweaked the comment for the AddRedirectToHttpsPermanent method to make it consistent.`. This is the only other overload in this class which mentions a status code, so it felt like the two should be aligned.

I can revert the change to this comment, and alter the comment for `AddRedirect` to be simply `,with returning a 302` (without the ` - Found status code`), which would then make it consistent with the `AddRedirectToHttpsPermanent` comments if that is preferable.

Reply:
Changing the format is fine, but you kept part of the old comment.

Reply:
Yes, that is intentional? It just adds the text version of what the status code represents, in addition to the numeric status code.

Reply:
Ok, if it's intentional then `status` -> `Status`.
Bit I think you should just remove `status code for permanently redirected.` now.

Reply:
Oh wow, I completely missed that the xml comment for the permanent option was carried on to the next line, my mistake. I have reverted the change to the permanent redirect option, and now aligned the 302 option to match what the permanent one does.